src/datasets/active_profiles.md

```diff
@@ -0,0 +1,140 @@
+# Intro
+
+The `active_profiles` dataset gives client-level estimates of whether a profile
+is still an active user of the browser at a given point in time, as well as probabilistic forecasts
+of the client's future activity. These quantities are estimated by a model that attempts to infer
+and decouple a client's latent propensity to leave Firefox and become inactive, as well as their
+latent propensity to use the browser while still active.
+
+
+# Model
+The model generates predictions for each client by looking at just the recency and frequency of a
+client's daily usage within the previous 90 day window. Usage is defined by the daily level binary
+indicator of whether they show up in `clients_daily` on a given day.
+
+The table contains columns related to these quantities:
+
+- `submission_date`: day marking the end of the 90 day window
+- `min_day`: first day in the window that the client was seen
```

Comment:
is it possible then that the min_day could be 2 days before the 90 day window closes? or do you select min day to be at least 90 days ago (so that there was a chance to come back)? were there particular dates you looked at overall for this analysis?

Reply:
> is it possible then that the min_day could be 2 days before the 90 day window closes? 

It could be anywhere in the window, from the first to last day. I'll add a note explaining that.

> were there particular dates you looked at overall for this analysis?

The model is continuously running (so today it's looking at data in the past 90 days including June, July, August, and September). When I was developing the model I was looking at different date intervals, but don't remember which. Did you mean that or something else?

Reply:
yeah, noting how far back the model goes might be good to add